entrypoint.sh

```diff
@@ -3,7 +3,7 @@
 while :
 do
 	DATE=$(date)
-	/opt/zap2xml.pl "$OPT_ARGS"
+	/opt/zap2xml.pl $OPT_ARGS
```

Comment:
I don't see an issue with this. Perhaps you can elaborate with your description (screenshots or logs)? Usually quotes are beneficial for environment variables for expanding properly.

Reply:
Here are the following logs of the problem I'm running into:

Attaching to zap2xml
zap2xml  | zap2xml (2018-12-01)
zap2xml  | Command line: /opt/zap2xml.pl -I -D -C /config/.zap2xmlrc -o /xmltv/xmltv.xml
zap2xml  | Unknown option:  
zap2xml  | Unknown option: -
zap2xml  | Unknown option:  
zap2xml  | Unknown option: -
zap2xml  | Unable to login: Unspecified username or password.
zap2xml  | Last run time: Wed Feb 12 22:08:35 UTC 2025
zap2xml  | Will run in 43200 seconds

Reply:
Oh how interesting... It's splitting it (it looks like).

What's your `OPT_ARGS` in your `docker-compose.yaml`?

Reply:
environment:
      OPT_ARGS: >-
        -I -D -C /config/.zap2xmlrc -o /xmltv/xmltv.xml

Reply:
Maybe instead of removing the quotes, we change this to:

```suggestion
	eval /opt/zap2xml.pl "$OPT_ARGS"
```

Reply:
Yes that works. Updated. Feel free to make the change directly yourself instead of merging if that's more convenient.